content/en/docs/reference/command-line-tools-reference/kube-proxy.md

```diff
@@ -15,6 +15,23 @@ environment variables specifying ports opened by the service proxy. There is an
 addon that provides cluster DNS for these cluster IPs. The user must create a service
 with the apiserver API to configure the proxy.
 
+Use-cases
+
+•	You can make your own proxy with custom behavior (antrea, cilliim, calico? and leverage from the ideas of the implementation of kube-proxy. 
```

Comment:
Change the first character to "*" please :)

Also, is this a question or a conclusion? I guess we can remove the examples of the CNIs implementations and leave something just like "You can make your own proxy [...] (without the thing between parenthesis)

Reply:
Is there in any other reference docs Use-cases? I'm trying to figure out what's the point of having this here :)